website/docs/r/lakeformation_resource_lf_tags.html.markdown

```diff
@@ -0,0 +1,134 @@
+---
+subcategory: "Lake Formation"
+layout: "aws"
+page_title: "AWS: aws_lakeformation_resource_lf_tags"
+description: |-
+    Manages an attachment between one or more LF-tags and an existing Lake Formation resource.
+---
+
+# Resource: aws_lakeformation_resource_lf_tags
+
+Manages an attachment between one or more existing LF-tags and an existing Lake Formation resource.
+
+## Example Usage
+
+### Database Example
+
+```terraform
+resource "aws_lakeformation_lf_tag" "example" {
+  key    = "right"
+  values = ["abbey", "village", "luffield", "woodcote", "copse", "chapel", "stowe", "club"]
+}
+
+resource "aws_lakeformation_resource_lf_tags" "example" {
+  catalog_id = data.aws_caller_identity.current.account_id
+
+  database {
+    name = aws_glue_catalog_database.example.name
+  }
+
+  lf_tag {
+    key   = aws_lakeformation_lf_tag.example.key
+    value = "stowe"
+  }
+}
+```
+
+### Multiple Tags Example
+
+```terraform
+resource "aws_lakeformation_lf_tag" "example" {
+  key    = "right"
+  values = ["abbey", "village", "luffield", "woodcote", "copse", "chapel", "stowe", "club"]
+}
+
+resource "aws_lakeformation_lf_tag" "example2" {
+  key    = "left"
+  values = ["farm", "theloop", "aintree", "brooklands", "maggotts", "becketts", "vale"]
+}
+
+resource "aws_lakeformation_resource_lf_tags" "example" {
+  catalog_id = data.aws_caller_identity.current.account_id
+
+  database {
+    name = aws_glue_catalog_database.example.name
+  }
+
+  lf_tag {
+    key   = "right"
+    value = "luffield"
+  }
+
+  lf_tag {
+    key   = "left"
+    value = "aintree"
+  }
```

Comment:
what happens if you assign the same key twice but with diferent values, e.g.

```terraform
lf_tag {
  key   = "left"
  value = "luffield"
}

lf_tag {
  key   = "left"
  value = "aintree"
}
```

Does the LF api catch this and error, or will it silently pick one?

Reply:
It catches the error:

```
Error: creating AWS Lake Formation Resource LF Tags ({
          LFTags: [{
              TagKey: "tf-acc-test-1974583174579168682-2",
              TagValues: ["vale"]
            },{
              TagKey: "tf-acc-test-1974583174579168682",
              TagValues: ["luffield"]
            },{
              TagKey: "tf-acc-test-1974583174579168682",
              TagValues: ["woodcote"]
            }],
          Resource: {
            TableWithColumns: {
              ColumnNames: ["event","timestamp"],
              DatabaseName: "tf-acc-test-1974583174579168682",
              Name: "tf-acc-test-1974583174579168682"
            }
          }
        }): InvalidInputException: Provided list of tags contains duplicate tag key
```